Add Arm64 related documents, add more content into debug.md (#682)
kubevirt/user-guide: docs/operations/operations_on_Arm64.md

> # Operations

Reviewer: ```suggestion
# Arm64 Operations
```

Right now, the menu bar on the left for all Operations just says "Operations" for this file, which is misleading.

This page summarizes all operations that are not supported on Arm64.

## Hotplug Network Interfaces

> Hotplug Network Interfaces does not support on Arm64, because the image ghcr.io/k8snetworkplumbingwg/multus-cni:snapshot-thick does not support for Arm64 platform. For more information please refer to https://github.com/k8snetworkplumbingwg/multus-cni/pull/1027.

Reviewer: ```suggestion
Hotplug Network Interfaces are not supported on Arm64, because the image ghcr.io/k8snetworkplumbingwg/multus-cni:snapshot-thick does not support the Arm64 platform. For more information please refer to https://github.com/k8snetworkplumbingwg/multus-cni/pull/1027.
```

Checking my understanding: the image does not support Arm64?

Author: Yes, you are right.

## Hotplug Volumes

> Hotplug Volumes does not support on Arm64, because Containerized Data Importer is not supported on Arm64 for now.

Reviewer: ```suggestion
Hotplug Volumes are not supported on Arm64, because the Containerized Data Importer is not supported on Arm64 for now.
```

## Hugepages support

> Hugepages feature does not support on Arm64. The hugepage mechanism is different from X86_64 and Arm64. Now we only verify KubeVirt on 4k pagesize system.

Reviewer: ```suggestion
The hugepages feature is not supported on Arm64. The hugepage mechanism differs between x86_64 and Arm64. Now we only verify KubeVirt on 4k pagesize systems.
```

## Containerized Data Importer

For now, we have not supported this project on Arm64, but it is in our plan.

## Export API

> We partially support it on Arm64 platform. As CDI has not supported yet, the export of DataVolumes and MemoryDump are not support on Arm64.

Reviewer: ```suggestion
Export API is partially supported on the Arm64 platform. As CDI is not supported yet, the export of DataVolumes and MemoryDump are not supported on Arm64.
```

Author: > As CDI is not supported yet,

As CDI has not been supported yet?

## Virtual machine memory dump

> As explained before, this feature does not support on Arm64.

Reviewer: ```suggestion
As explained above, MemoryDump requires CDI, and is not yet supported on Arm64.
```
